README.md

```
@@ -2,15 +2,15 @@
 [![Copier](https://img.shields.io/endpoint?url=https://raw.githubusercontent.com/copier-org/copier/master/img/badge/badge-black.json)](https://github.com/copier-org/copier)
 [![Actions status](https://www.github.com/LabAutomationAndScreening/copier-aws-organization/actions/workflows/ci.yaml/badge.svg?branch=main)](https://www.github.com/LabAutomationAndScreening/copier-aws-organization/actions)
 [![Open in Dev Containers](https://img.shields.io/static/v1?label=Dev%20Containers&message=Open&color=blue)](https://vscode.dev/redirect?url=vscode://ms-vscode-remote.remote-containers/cloneInVolume?url=https://www.github.com/LabAutomationAndScreening/copier-aws-organization)
-
+[![OpenIssues](http://isitmaintained.com/badge/open/LabAutomationAndScreening/copier-aws-organization.svg)](http://isitmaintained.com/project/LabAutomationAndScreening/copier-aws-organization)
 
 # Usage
```

markdownlint-cli2 (0.18.1)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)